src/Makefile.am

```diff
@@ -1,2 +1,2 @@
-noinst_LIBRARIES = libknot.a
-libknot_a_SOURCES = knot.c
```

Comment:
Squash this commit into "Renaming KNoT protocol files" to avoid breaking the build.